examples/smart_contracts/v2/python/account_delegation.py

```diff
@@ -20,61 +19,28 @@ def wait_for_confirmation(client, txid):
         txid, txinfo.get('confirmed-round')))
     return txinfo
 
-# Read a file
-def load_resource(res):
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    path = os.path.join(dir_path, res)
-    with open(path, "rb") as fin:
-        data = fin.read()
-    return data
-
 try:
-
     # Create an algod client
-    # algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-    # algod_address = "http://localhost:4001"
-    algod_token = "6b3a2ae3896f23be0a1f0cdd083b6d6d046fbeb594a3ce31f2963b717f74ad43"
-    algod_address = "http://127.0.0.1:54746"
-    # algod_token = "<algod-token>"
-    # algod_address = "<algod-address:port>" 
-    # receiver = "<receiver-address>" 
+    algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    algod_address = "http://localhost:4001"
+
     receiver = "ATTR6RUEHHBHXKUHT4GUOYWNBVDV2GJ5FHUWCSFZLHD55EVKZWOWSM7ABQ" 
```

Comment:
Is this TestNet faucet?

Reply:
I don't know. It was just in the code already. I did not change.